airflow/providers/airbyte/operators/helper.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+
+def map_airbyte_type(type: str, format: Optional[str]) -> str:
+    if type == "string" and format == "date-time":
+        return "timestamp"
+
+    if type == "string" and format == "date":
+        return "date"
+
+    if type == "string":
+        return "string"
```

Comment:
Maybe we should go with something like below or create some kind of dict with mapping?
```suggestion
    if type in ("string", "boolean", "object", "array"):
        return type
```

Reply:
I think current implementation is easy to read :)